rainbow-me/rainbowkit
# chore: upgrade `next`, `eslint-config-next` and `next-intl`

## Changes
- `next` to `^14.1.0`
- `eslint-config-next` to `^14.1.0`
- `next-intl` to `^3.9.1`
- Updated `site` and `with-next-app-i18n` to use the latest `next-intl` setup

## What to test
1. Make sure nothing breaks in `site`, `examples` and `packages` with the latest next updates
2. Run `pnpm build` and make sure nothing breaks either
3. If all works you're set ✨
